File: Cpp/LW3/lr3 z1/lr3 z1/student.cpp
#include "student.h"
#include <iostream>
using namespace std;

student::student(string pib, string date, string city, string phone, string department, int course)
    : persona(pib, date, city, phone), department(department), course(course) {
}

void student::info() const {
    cout << "Студент: " << pib << endl;
    cout << "Дата народження: " << date << " (Вiк: " << age() << ")" << endl;
    cout << "Заклад: " << city << endl;
    cout << "Телефон: " << phone << endl;
    cout << "Вiддiлення: " << department << endl;
    cout << "Курс: " << course << endl;
    cout << "-----------------------------\n";
}

File: Csharp/LW8/lr8_1dll/lr8_1dll/student.cs
using lr8_1dll;
using System;
using System.Security.Policy;

namespace lr8_1dll
{
    public class Student : Persona
    {
        public string Department { get; set; }
        public int Course { get; set; }

        public Student(string pib, string date, string city, string phone,
                       string department, int course)
            : base(pib, date, city, phone)
        {
            Department = department;
            Course = course;
        }

        public override void Info()
        {
            Console.WriteLine("Студент:");
            Console.WriteLine($"ПІБ: {PIB}");
            Console.WriteLine($"Кафедра: {Department}");
            Console.WriteLine($"Курс: {Course}");
            Console.WriteLine($"Місто: {City}");
            Console.WriteLine($"Телефон: {Phone}");
        }
    }
}